phpstan-baseline.neon

```diff
@@ -410,6 +405,21 @@ parameters:
 			count: 1
 			path: tests/functional/Teradata/ExportTest.php
 
+		-
+			message: "#^Parameter \\#1 \\$callback of function array_map expects \\(callable\\(array\\{Key\\: string\\}\\|Google\\\\Cloud\\\\Storage\\\\StorageObject\\|MicrosoftAzure\\\\Storage\\\\Blob\\\\Models\\\\Blob\\)\\: mixed\\)\\|null, Closure\\(Google\\\\Cloud\\\\Storage\\\\StorageObject\\)\\: string given\\.$#"
+			count: 1
+			path: tests/functional/Teradata/ToFinal/IncrementalImportTest.php
```

Comment:
co tohle, da se to odstranit?

Reply:
dal jsem tomu hodinu a neprisel jsem na to jak. Imho je to spis bug v phpstanu